tests/mypy/success_strict_bool.py

```diff
@@ -0,0 +1,5 @@
+from pydantic import BaseModel, StrictBool
+
+
+class Flags(BaseModel):
+    strict_bool: StrictBool = False
```

Comment:
surely all success cases can go in `tests/mypy/success.py`? since everything should pass.

I think move this in there. If we think successful tests should be split by what's being tested, we should split each test and change `expected_successes` to include all files in that directory that match `fail*.py`

Reply:
I went to put it there but it felt like that file was growing out of control. I’m fine with that though if you are!

Other strict types don’t need it since they are subclasses of the non-strict type. This one needs it because of the weird you-can’t-subclass-bool thing in python.

Reply:
> I went to put it there but it felt like that file was growing out of control. I’m fine with that though if you are!

I think put it there for now, if more changes are needed let's do that separately.

Reply:
Done